Resources/Prototypes/_Arcadis/Entities/Objects/Weapon/Battery/battery_guns.yml

```
@@ -231,3 +231,80 @@
     wideAnimationRotation: 135
   - type: DamageOtherOnHit
     staminaCost: 5
+
+- type: entity
+  name: CRT-20 Laser Assault Rifle
+  parent: [BaseWeaponPowerCell, BaseGunWieldable]
+  id: WeaponLaserAssaultLaserRifleCR #LCP : Laser connection port
+  description: A laser assault rifle powered by a special battery, featuring a small LCP connection port for seamless integration, delivering precise, high-powered energy firepower in combat.
```

```suggestion
  description: A laser assault rifle powered by a special battery, featuring a small laser connection port for seamless integration. Designed to deliver precise, high-powered energy firepower in combat.
```
“connection port” becomes redundant if LCP stands for laser connection port